addons/hr/views/hr_department_views.xml

```diff
@@ -75,7 +75,8 @@
                                 <div class="container o_kanban_card_content" t-if="!selection_mode">
                                     <div class="row">
                                         <div class="col-6 o_kanban_primary_left">
-                                            <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action">Employees</button>
+                                            <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action"
+                                                    context="{'searchpanel_default_department_id': active_id}">Employees</button>
```

Comment:
I would rather fix this like in https://github.com/odoo/odoo/pull/36669/commits/dee7f91568a2d470aed4881f8542cb2f08dce216

Reply:
I totally agree